tuxemon/event/conditions/party_size.py

```diff
@@ -21,7 +22,8 @@ class PartySizeCondition(EventCondition):
             is party_size <operator>,<value>
 
     Script parameters:
-        operator: One of "equals", "less_than" or "greater_than".
+        operator: Numeric comparison operator. Accepted values are "less_than",
```

Comment:
Can we change this to use normal operators? (> < = etc).

Keep it backwards compatible to avoid breaking any in progress work.

Reply:
Done, backwards compatible, but I noticed that there aren't many greater_than, etc.
It was affected also has_item condition (action)